src/Event/ErrorEvent.php

```diff
@@ -14,27 +14,13 @@
 
 class ErrorEvent extends Event
 {
-    /**
-     * @var Exception
-     */
-    protected $error;
-
     /**
      * @param Exception $error
+     * @param string    $request_name
      */
-    public function __construct($error)
-    {
-        $this->error = $error;
-
-        parent::__construct(array('error' => $error));
-    }
-
-    /**
-     * @return Exception
-     */
-    public function getContext()
+    public function __construct($error, $request_name)
```

Comment:
looks like the README needs to be updated

Reply:
Thanks, I've updated it here 766412e